### ISSUE TITLE
Error: E_FONT_FAMILY_INCONSISTENT

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#797]
This error occurs with https://vivliostyle.vercel.app/#src=https://ethantw.github.io/Han/latest/

The error occurs since v2.10.0:
https://vivliostyle.github.io/viewer/v2.10.0/#src=https://ethantw.github.io/Han/latest/em.html
~v2.9.1 was ok~:
https://vivliostyle.github.io/viewer/v2.9.1/#src=https://ethantw.github.io/Han/latest/em.html

~This bug would be a side effect of~:

- #767

**Comment by MurakamiShinyu** [comment]
"v2.9.1 was ok ..." was wrong. This is a very old bug on `@font-face` implementation. I made a simple test that can reproduce this error even with very old Vivliostyle versions.

```css
@font-face {
  font-family: "FontA";
  src: local("Arial");
}
@font-face {
  font-family: "FontB";
  src: local("Times New Roman");
}
@font-face {
  font-family: "FontA";
  font-weight: bold;
  src: local("Arial Bold");
}
@font-face {
  font-family: "FontB";
  font-weight: bold;
  src: local("Arial Bold");
}
```

https://jsbin.com/zakisolocu/edit?html,output
 
Test with Vivliostyle Viewer:
- version 20016.1: https://vivliostyle.github.io/viewer/2016.1/vivliostyle-viewer.html#x=https://jsbin.com/zakisolocu
- latest canary: https://vivliostyle.vercel.app/#src=https://jsbin.com/zakisolocu

**Comment by MurakamiShinyu** [comment]
Fixed by #798

Test: https://vivliostyle-git-fix-issue797-vivliostyle.vercel.app/#src=https://ethantw.github.io/Han/latest/
